src/components/ccsds_command_depacketizer/gen/templates/assembly/name_ccsds_cosmos_telemetry.txt

```diff
@@ -0,0 +1,32 @@
+{% for id, packet in packets.items() %}
+TELEMETRY {{ name|upper }} {{ packet.name|upper }} BIG_ENDIAN "{{ packet.description }}"
+{# The ccsds packets header #}
+{# This header assumed by the Ccsds_packets_Depacketizer component #}
```

Comment:
Same note here about the hard coded CCSDS header

Reply:
CCSDS header definitions applied in [b7b8f71](https://github.com/lasp/adamant/pull/21/commits/b7b8f71fd9f2b276587e961b0a1f2253a44f7bd4), secondary header definitions to be applied in a future commit.